scripts/wrap.sh

```diff
@@ -1,27 +1,79 @@
 #!/bin/bash
+
+set -e  # Exit on any error
+
+# Determine rootdir based on our script location
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+ROOTDIR="$(dirname $DIR)"
+
+# Detect luamin binary
+LUAMIN="luamin"
+if ! which $LUAMIN 1> /dev/null; then
+    LUAMIN="${ROOTDIR}/scripts/node_modules/.bin/luamin"
```

Comment:
By the way, the reason I did not delegate to the installed luamin is that we *might* take changes to Luamin for custom minification.  This negates the need to keep the build system machine and the dev machines in sync because they both use the same sources, always.